lib/valkyrie/persistence/fedora.rb

```diff
@@ -1,8 +1,13 @@
 # frozen_string_literal: true
-#
 module Valkyrie::Persistence
   # Implements the DataMapper Pattern to store metadata into Fedora
   module Fedora
+    # Deprecation to allow us to make pg an optional dependency
```

Comment:
Forgive the nitpicking, but I think this was a copy paste - should be ldp no pg?